src/main/java/org/gatorgradle/config/GatorGradleConfig.java

```diff
@@ -185,6 +186,27 @@ private Command makeCommand(String path, String line, boolean pure) {
     return cmd;
   }
 
+  private String readFile(String fileName) throws IOException {
```

Comment:
This utility method should not be in `GatorGradleConfig`, but rather in a class in `util`. And actually, instead of a utility method to read a file into a string, you should use `String.join("\n", Files.readAllLines(Paths.get(fileName)))` in the place you call this method, which will be both faster, simpler, and easier to understand, while using newer API calls.